auth_password_settings/views/res_config_view.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data>
+
+        <record id="view_general_configuration" model="ir.ui.view">
+            <field name="name">base.config.settings.password.settings</field>
+            <field name="model">base.config.settings</field>
+            <field name="inherit_id" ref="base_setup.view_general_configuration"/>
+            <field name="arch" type="xml">
+                <xpath expr="//label[@string='Email']/.." position="before">
+                    <group>
+                        <label for="id" string="Password Strength"/>
+                        <div>
+                            <p>
+                                <label string="This section allows you to determine password complexity (the effectiveness of a password in resisting guessing and brute-force attacks). To do that," /><br/>
+                                <label string="* use the Minimum password length to determine how long the password should be to be considered strong, By default, the password should be at least 6 characters long."/><br/>
```

Comment:
I'm not sure if we should keep the "By default, the password should be at least 6 characters long."